hosts/galileo/__init__.py

```diff
@@ -0,0 +1,395 @@
+                   spk='reconstructed', gapfill=True,
+                   mst_pck=True, irregulars=True):
+        """Intialize the Galileo mission internals.
+
+        After the first call, later calls to this function are ignored.
+
+        Input:
+            ck,spk      Used to specify which C and SPK kernels are used.:
+                        'reconstructed' for the reconstructed kernels (default);
+                        'predicted' for the predicted kernels;
+                        'none' to allow manual control of the C kernels.
+            planets     A list of planets to pass to define_solar_system. None
+                        or 0 means all.
+            asof        Only use SPICE kernels that existed before this date;
+                        None to ignore.
+            gapfill     True to include gapfill CKs. False otherwise.
+            mst_pck     True to include MST PCKs, which update the rotation
+                        models for some of the small moons.
+            irregulars  True to include the irregular satellites;
+                        False otherwise.
+        """
+        if Galileo.initialized:
+            return
+
+        (ck, spk) = ('NONE', 'NONE')
+
+        # Define some important paths and frames
+        Body.define_solar_system(Galileo.START_TIME, Galileo.STOP_TIME,
+                                 asof=asof,
+                                 planets=planets,
+                                 mst_pck=mst_pck,
+                                 irregulars=irregulars)
+
+        _ = oops.path.SpicePath('GLL', 'JUPITER')
+
+        spicedb.open_db()
+
+        spk = spk.upper()
+        if spk == 'NONE':
+
+            # This means no SPK will ever be loaded; handling is manual
+            Galileo.initialize_kernels([], Galileo.SPK_LIST)
+            Galileo.SPK_LOADED = np.ones(Galileo.MONTHS, dtype='bool')
+        else:
+            kernels = spicedb.select_spk(-77, name='CAS-SPK-' + spk,
+                                              time=(Galileo.START_TIME,
+                                                    Galileo.STOP_TIME),
+                                              asof=asof)
+            Galileo.initialize_kernels(kernels, Galileo.SPK_LIST)
+
+        ck = ck.upper()
+        if ck == 'NONE':
+
+            # This means no CK will ever be loaded; handling is manual
+            Galileo.initialize_kernels([], Galileo.CK_LIST)
+            Galileo.CK_LOADED = np.ones(Galileo.MONTHS, dtype='bool')
+        else:
+            kernels = spicedb.select_ck(-77, name='CAS-CK-' + ck,
+                                             time=(Galileo.START_TIME,
+                                                   Galileo.STOP_TIME),
+                                             asof=asof)
+            Galileo.initialize_kernels(kernels, Galileo.CK_LIST)
+
+        # Load extra kernels if necessary
+        if gapfill and ck not in ('PREDICTED', 'NONE'):
```

Comment:
This code is not appropriate for Galileo

Reply:
removed